packages/header-component/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@debtcollective/dc-header-component",
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
Not sure in which moment I prone this to change. Besides, it is pretty out dated.